pages/user guides/discover_nmap.md

```diff
@@ -8,58 +8,53 @@ permalink: discover-nmap.html
+2. Connect Tidal Tools and your Tidal Migrations account with [tidal login](https://guides.tidalmg.com/tidal-tools.html#login).
+3. Run Nmap with the flags of your choosing and save the output to an XML file. For example,
     ```
-    sudo nmap -sS -p80,443,8080,8443,1433,1521,27017 <ip-address> -oX my-network.xml
+    sudo nmap -sV -p80,443,8080,8443,1433,1521,27017 <ip-address/range> -oX my-network.xml
     ```
-    _Note: the **-sS** flag will enable TCP SYN port scan and the **-oX** specifies the output as a XML file. [Want more scanning options?](#nmap-usage)_
-    {% include tip.html content="Be sure to replace <ip-address> with the CIDR range for the network you would like to scan, ex. `10.0.0.0/24`" %}
+    [Go here to install Nmap](#installing-nmap).
+    
+    _Note: the **-sV** flag will attempt to determine the version of the service running on port and the **-oX** specifies the output as an XML file. [Want more scanning options?](#nmap-usage)_
 
+    {% include tip.html content="Be sure to replace <ip-address/range> with the CIDR range for the network you would like to scan, ex. `10.0.0.0/24`" %}
+ 
 4. Run this Tidal Tools command to upload your previously generated Nmap output to your Tidal Migrations account 
 ```
 tidal sync nmap <filename.XML> 
 ``` 
+5. Head over to your Tidal Migrations account! (https://<account_name>.tidalmg.com/#/labs/host_discovery)
  
-5. Head over to your Tidal Migrations account!
- 
-
-
 ## Installing Nmap
-
 Nmap ("Network Mapper") is a free and open source utility for network discovery and security auditing. Many systems and network administrators also find it useful for tasks such as network inventory, managing service upgrade schedules, and monitoring host or service uptime.
- 
-You can find all the documentation in its official site [Nmap](https://nmap.org/) and instructions for how to download based on your environment [download](https://nmap.org/download.html)
+
+You can find all the documentation and instructions on how to download [Nmap](https://nmap.org/) to your environment on the official site.
 
 
 ## Nmap usage
 {% include callout.html content="Note: Currently, Tidal Migrations supports the collection of Hosts (IP addresses), PTR records, their open ports, the ports status, the port protocol (TCP/UDP), and the services running in the port, including the version."  type="primary"%}
+Nmap offers a wide range of utilities and commands, such as Port scanning, Host discovery, Service and version detection to name a few. Here are some basic examples for how to do some nmap scanning.
 
-Nmap offers a wide range of utilities and commands, such as Port scanning, Host discovery, Service and version detection to name a few. Here some basic examples for how to do some nmap scanning.
-_NB: Some commands required sudo privilege._
+_NB: Some commands require super-user privilege._
  
-#### Target Specification
+### Target Specification
  
 ```
 nmap 192.168.1.1-254                # Scan a range
  
```

Comment:
This empty line is “visible” with the final HTML output

![image](https://user-images.githubusercontent.com/290451/100791706-acf5e900-342a-11eb-85f6-c080f38b046f.png)